### PR TITLE
Please add my difficult solution.

### PR DESCRIPTION
**Pull request by RudiBoshoff**
I have also added a little clarification about which "settings" to select as I was searching in settings of Github and not within the specific repository of where my project was uploaded.

```
This is a PR template, If you are adding a solution link to the curriculum leave this as is. If you are not, then delete it and write the message you wish.
Thank you,
The Odin Project team.

```

### CODE REVIEW

File: web_development_101/project_html_css.md [hunk 1]
```diff
@@ -77,7 +77,7 @@ Github allows you to publish web projects directly from a github repository.  Do
 There are a couple of ways to go about doing this, but the simplest is this:
 
 - make sure that the main html file of your project is called `index.html`.  If it is not, you will need to rename it.
-- go to your github repo on the web and click the __Settings__ button
+- go to your online github repository for the specific project eg.`https://github.com/YourName/google-homepage` and click the __Settings__ button for that repository.
```
Comment (KevinMulhern) [inline web_development_101/project_html_css.md]: Hi @RudiBoshoff, thanks for this PR. Do you think these updated instructions are needed? The extra words make this a little more unreadable to me.

Comment (RudiBoshoff) [reply web_development_101/project_html_css.md]: I guess not, but I personally did struggle to realize that "github repo" was specifying the repository of the file I was required to upload and not my overall github account

File: web_development_101/project_html_css.md [hunk 2]
```diff
@@ -1413,7 +1413,7 @@ Submit a link below to the github repo with your files in it by using a pull req
 * [Faris Ibrahim's Solution (Easy)](https://github.com/procusr/google-homepage) - [View in Browser](https://procusr.github.io/google-homepage/)
 * [Giorgi Shengelaia's Solution (Easy)](https://github.com/supuun/Google_Clone) - [View in Browser](https://supuun.github.io/Google_Clone/)
 * [3lux's solution (Easy)](https://github.com/3lux/google-homepage) - [View in browser](https://3lux.github.io/google-homepage/)
-* [Rudi Boshoff's Solution (Easy)](https://rudiboshoff.github.io/google-homepage/) - [View in browser](https://rudiboshoff.github.io/google-homepage/)
+* [Rudi Boshoff's Solution (Easy)](https://rudiboshoff.github.io/google-homepage/) - [View in browser](https://rudiboshoff.github.io/google-homepage/) - [Rudi Boshoff's Solution (Difficult)](https://github.com/RudiBoshoff/google-search-results) - [View in Browser](https://rudiboshoff.github.io/google-search-results/)
```
Comment (RudiBoshoff) [inline web_development_101/project_html_css.md]: Thank you for your response on my suggestion about the instructions. Please could you also accept my changes to add my difficult solution to the overall document. Thank you again

Comment (mindovermiles262) [reply web_development_101/project_html_css.md]: Hi @RudiBoshoff 

To add your solution to this without the instructions you'll have to open a new pull request with just the addition.

**Pull request closed without merging**